tests/Backend/Synapse/CreateTableTest.php

```diff
@@ -103,4 +103,62 @@ public function testCreateTableWithCrlfLineEndings()
         $importedData = $this->_client->getTableDataPreview($tableId);
         $this->assertCount(5, Client::parseCsv($importedData));
     }
+
+    public function testCreateTypedTable()
```

Comment:
tady není žádný assert bo se mi z toho jobu nedaří nic dostat abych to mohl assertnout. Results v tabulce bi_jobs zůstává prázdné